apps/hash-frontend/src/shared/layout/layout-with-sidebar/sidebar.tsx

```diff
@@ -56,6 +58,46 @@ export const PageSidebar: FunctionComponent = () => {
 
   const { draftEntities } = useDraftEntities();
 
+  /**
+   * @todo H-2664 – remove this
+   */
+  if (frontendUrl !== "http://localhost:3000") {
+    enabledFeatureFlags.workers = false;
+  }
```

Comment:
Is this neccessary? The feature flag will be disabled in production (`workers` won't be present for any users in production), and the alice user is automatically seeded with all feature flags enabled so it should be automatically enabled in development when logged in as that user.
```suggestion
```

Reply:
I guess not, no